src/org/loklak/data/ElasticsearchClient.java

```diff
@@ -988,4 +990,65 @@ public TermsBuilder getClassifierAggregation(String classifierName) {
                 AggregationBuilders.sum("sum_probability").field(probabilityField)
             );
     }
+
+
+    public HashMap<String, HashMap<String, Double>> classifierScore(String index, String classifierName, ArrayList<String> classes, String startDate, String endDate) {
```

Comment:
The top level data aggregation restricts the time to the desired range. The sub-aggregations added to it help perform the summation, averaging and document count jobs.